dataland-frontend/src/components/forms/parts/fields/StringBaseDataPointFormField.vue

```diff
@@ -1,16 +1,20 @@
 <template>
   <div class="form-field">
-    <FormKit v-model="baseDataPoint" type="group" name="dataSource">
+    <FormKit v-model="baseDataPoint" type="group" ignore="true">
       <UploadDocumentsForm
         @updatedDocumentsSelectedForUpload="handleDocumentUpdatedEvent"
         ref="uploadDocumentsForm"
         name="name"
         :more-than-one-document-allowed="false"
         :file-names-for-prefill="fileNamesForPrefill"
       />
+    </FormKit>
+
+    <FormKit v-if="isValidFileName(isMounted, documentName)" type="group" name="dataSource">
       <FormKit type="hidden" name="fileName" v-model="documentName" />
-      <FormKit type="hidden" name="fileReference" v-model="documentReference" />
+      <FormKit type="hidden" name="fileReference" :modelValue="documentReference" />
```

Comment:
Why use `modelValue` for `fileReference` but not for `fileName`? They basically work the same.

Reply:
Not my code.
Git says @pioSko2

Reply:
Does `:modelValue` work as two-way binding? Will it also update the value if it changes during Edit mode?

Reply:
I think it is a one way binding. But this is fine since this field is hidden and thus can not manipulated by the user.

Reply:
Oh. On edit we need a two way binding actually I think.

Reply:
is this addressed or something needs to be changed?

Reply:
Oh, right. I didn't notice that. All other components with dataSource have the fileReference computed based on the fileName. So a `:modelValue` is sufficient there.
I'll change and test.

Reply:
I changed it to v-model. In this case it seems more appropriate.

Reply:
Ah. Now I get your point.